docs/manual/manual_stow.rst

```diff
@@ -0,0 +1,310 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_stow_position:
+
+***********************************************
+Stow Position
+***********************************************
+The stow position feature allows a Vertiq module to return to a configurable position on a transition from armed to disarmed, on timeouts, or when given an 
+explicit command to stow. This can be useful for holding propellers in an aerodynamic position, or preparing vehicles for storage. Users can control what 
+this position is, when the module should attempt to move into the stow position, how aggressively it moves to the stow position, and the module’s behavior 
+once it reaches the stow position. Note that incoming commands can cause interruptions to the stow process, as detailed in :ref:`interrupting_stow`.
+
+Module Support
+===============
+
+Speed Modules
+**************
+
+.. table:: Speed Module Support for Stow Position
+
+	+-------------+------------------------------------+
+	| Module      | Stow Position Support              |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support stow as a separate feature. Rather, they are intended to primarily serve as position modules, so controlling them to move to any desired

```

Comment:
I think it is valuable to explain why its not supported, because I think customers are going to come here and wonder why they can't get stow on their servo modules, without realizing its redundant. A red X doesn't give them any of that context, all it tells them is that for some reason their servo module is missing some feature they may think they need